### DISCUSSION TITLE
can't use array in languages configuration

### DISCUSSION THREAD

**Discussion by maguzzi** [ppoffice/hexo-theme-icarus#1259] [Q&A] [answered]
Hi,
If I try to put an array in the language config (_config.yml), _hexo generate_ fails with this error:

`language: [en,es]`

```
TypeError: lang.toLowerCase is not a function
    at toMomentLocale (D:\marcoaguzzi_src\node_modules\hexo-theme-icarus\node_modules\hexo\lib\plugins\helper\date.js:87:15)

```
hexo: 7.0.0
node: 20.10.0

hexo icarus theme is at lastest version
am I missing something?

thanks!

**Comment by maguzzi** [comment]
I've tried some debugging, and the error is only non non-post pages (index, tags... so the input causing an error is the languages being an array).  
I've tried a fix, stating:
if lang.length>1 => lang=lang[0] 
doing so the error is prevented, but the message with the create, update, and read time gets wrong, such as:

4 minutos de lectura => 4 mintues de lectura (times are in english)

**Comment by ppoffice** [comment, marked as answer]
@maguzzi This should be fixed with https://github.com/ppoffice/hexo-theme-icarus/commit/49568ab9b08fba4e8702ffeef2a77c521a9d0544. However, Icarus currently cannot really take advantage of multiple languages set in `_config.yml`.

**Reply by maguzzi** [reply]
Thanks for the answer! I don't know the nodejs ecosystem and didn't really understand yet how to use a custom version of the icarus theme (or of a plugin) without referencing the one in the node_modules folder. Do you have any pointers I can follow, so that I can expertiment a little bit?
Thanks
Regards
M
